src/account/Account.ts

```diff
@@ -98,4 +100,77 @@ export class Account implements IAccount {
       );
     });
   }
+
+  /**
+   * This method aims to buy rolls on behalf of the sender.
+   *
+   * @param amount - The amount of rolls to be purchased
+   * @param fee - The fee to be paid for the transaction.
```

Comment:
We are targeting dev not used to build on blockchain. we need to be clear on for who the fees are :-)
```suggestion
   * @param fee - The fee to be paid for the transaction execution by the node.
```

Reply:
changed